charm/charmcraft.yaml

```diff
@@ -60,32 +51,24 @@ requires:
 
 config:
   options:
-    saml_sp_entity_id:
+    saml-sp-entity-id:
```

Comment:
Why hyphen instead of underscore?

Reply:
It is the new convention for `paas-app-charmer` projects